### DISCUSSION TITLE
Hide Taskbar when Firefox enters full-screen

### DISCUSSION THREAD

**Discussion by Allaman** [lawand-dot-io/taskbar#27] [Feature requests]
**Describe the bug**
Full screen YouTube video overlaps with taskbar

**To Reproduce**
1. Start taskbar
2. Watch a YouTube video in Firefox and go in fullscreen mode

**Expected behavior**
taskbar should be "hidden" in a full screen video

**Screenshots**
<img width="2304" alt="image" src="https://github.com/lawand-dot-io/taskbar/assets/12184268/d25e411d-0f27-4f39-a22a-32023c53f53c">

**Desktop (please complete the following information):**
 - Taskbar version 1.0.6
 - OS: [13.6
 - Chip M1 Pro

**Additional context**
Firefox 121

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the bug report. I am aware of this issue as I personally user Firefox as the main browser. This is happening because firefox doesn't handle being fullscreen the way that's native to macOS, so it's a bit difficult to detect that. I'll think about a way to solve this, but for the moment my temporary solution is that I use the "hide for now" feature. Have you seen that one? If you scroll down, taskbar is hidden and it shows up again when you move the mouse to the bottom of the screen

**Comment by Allaman** [comment]
Another Firefox fellow, nice! 😄 

"hide for now" works. I saw that feature and also "the scroll up/down" settings. I must admit that it wasn't clear to me. For the latter one, I ended up scrolling down my browser waiting for something to happen 😬 Maybe you want to implement a tooltip.

**Comment by lawand-dot-io** [comment]
Firefox v124 now uses the macOS fullscreen API, just like Safari and Chrome, so this issue should no longer happen.
